src/OpenEventSourcing.Azure.ServiceBus/OpenEventSourcing.Azure.ServiceBus.v3.ncrunchproject

```diff
@@ -0,0 +1,5 @@
+﻿<ProjectConfiguration>
```

Comment:
Is it worth adding all these ncrunch files to the .gitignore?

Reply:
Nope, the `*.ncrunchproject` should be in source control so that they can be shared, there may be cases where there are settings here which would prevent builds/tests running without them (at least for ncrunch users)